src/bundle.ts

```diff
@@ -0,0 +1,59 @@
+import { TranspiledFileWithSourceNode } from "./Transpile";
+import { SourceNode } from "source-map";
+
+const trimExtension = (fileName: string) => (fileName.endsWith(".ts") ? fileName.slice(0, -3) : fileName);
```

Comment:
There are a lot of other extensions TypeScript supports, we should trim all of them.
In refactor PR I've already moved `trimExt` from `Emit.ts` to `utils.ts` with `trimExtension` name.